services/player-service/domain/src/main/java/com/kjeldsen/player/domain/Team.java

```diff
@@ -0,0 +1,14 @@
+package com.kjeldsen.player.domain;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
```

Comment:
let's avoid using Data as it adds a lot of unnecessary stuff - I did added it many times but going forward let's reduce it's usage

Reply:
Ok, I'm gonna remove it just to avoid unnecessary equals and hash methods, for now I don't think we need them. Good point!!